client/src/components/Verejne/index.js

```diff
@@ -44,8 +57,9 @@ export default compose(
   connect(
     (state) => ({
       entities: entitiesSelector(state),
+      entitiesLength: entitiesSelector(state).length,
```

Comment:
You can use props.entities.length directly, no need for additional state

Reply:
no I can't since you override entities in withProps

Reply:
Oh, sorry I missed that.